### PR TITLE
HA service send and event_register for multiple addresses

### PR DESCRIPTION
**Pull request by farmio**
## Description

HA service `send` and `event_register` accept multiple addresses.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] I have performed a self-review of my own code
- [x] The documentation has been adjusted accordingly
- [x] The changes generate no new warnings
- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] The changes are documented in the changelog
- [x] The Homeassistant plugin has been adjusted in case of new config options

### CODE REVIEW

File: home-assistant-plugin/custom_components/xknx/__init__.py [hunk 1]
```diff
@@ -140,15 +140,21 @@
 SERVICE_XKNX_SEND_SCHEMA = vol.Any(
     vol.Schema(
         {
-            vol.Required(SERVICE_XKNX_ATTR_ADDRESS): cv.string,
+            vol.Required(SERVICE_XKNX_ATTR_ADDRESS): vol.All(
+                cv.ensure_list,
+                [cv.string],
```
Comment (marvin-w) [inline home-assistant-plugin/custom_components/xknx/__init__.py:145]: I guess this makes sure the list can contain one element of type string? (backwards compatibility)

**Review by marvin-w — APPROVED** [review]
Looks good to me, lovely!

File: home-assistant-plugin/custom_components/xknx/__init__.py (hunk 1 above)
Comment (farmio) [reply home-assistant-plugin/custom_components/xknx/__init__.py:145]: Yep 👍

**Comment by farmio** [comment]
added from HA upstream with #627

**Pull request closed without merging**